eox_core/settings/common.py

```diff
@@ -49,3 +49,7 @@ def plugin_settings(settings):
 
     # Sentry Integration
     settings.EOX_CORE_SENTRY_INTEGRATION_DSN = None
+    settings.EOX_CORE_SENTRY_IGNORED_ERRORS = [
+        'xmodule.exceptions.NotFoundError',
```

Comment:
Can we put this as an example on the sentry.py module docstring and remove it from here.

Then we can add it only for the SAAS on EOX_CORE_SENTRY_IGNORED_ERRORS

Reply:
I added that to the README